### ISSUE TITLE
BUG - Camera permissions are requested even when in audio only mode

### ISSUE THREAD

**Issue by james-cohen** [daily-co/react-native-daily-js#59]
# Expected behavior

Create a call object for an audio only call, microphone permissions only are requested

# Describe the bug (unexpected behavior)

Camera permissions are requested even thought video source is false

# Steps to reproduce

Create new audio only call object on an Android device as [described in guide](https://docs.daily.co/guides/products/audio-only#set-up-an-audio-only-app-with-daily):

```
      const newCallObject = Daily.createCallObject({
        videoSource: false,
        audioSource: true,
        startAudioOff: false,
      });
```
Camera permissions modal appears, sometimes before and sometimes after mic permissions

# System information 

 

* Device: Samsung Galaxy Z Flip 5
* OS, version: Android 14
* Browser, version: React Native

**Comment by james-cohen** [comment]
The issue may be due to the [checkPermissions method in DailyNativeUtils.java](https://github.com/daily-co/react-native-daily-js/blob/d8d0592ffd9f9b828189432e37b5a445efc2b9b1/android/src/main/java/com/daily/reactlibrary/DailyNativeUtils.java#L132) - both camera and mic permissions seem to be requested regardless

**Comment by filipi87** [comment]
Hi @james-cohen, thank you for reporting this. We were able to replicate the issue and plan to fix it before our next release. I’ll let you know once it's ready.

**Comment by filipi87** [comment]
This issue has been fixed in `react-native-daily-js` version [0.70.0](https://github.com/daily-co/react-native-daily-js/releases/tag/react-native-daily-js-2024-10-31-0.70.0).

In order for it to properly work, the `AndroidManifest.xml`:
- Should **NOT** include the permission for the foreground service to access camera:
  - `<uses-permission android:name="android.permission.FOREGROUND_SERVICE_CAMERA"/>`
- The `foregroundServiceType` should **NOT** include `camera`.
